.env.development

```
@@ -6,4 +6,4 @@ VITE_APP_ENV = 'development'
 
 # 管理系统模板/开发环境
 # VITE_APP_BASE_URL = 'http://localhost:80'
-VITE_APP_BASE_URL = 'http://192.168.110.68:6088'
+VITE_APP_BASE_URL = 'http://192.168.110.68:8080'
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)